crates/anvil-polkadot/src/substrate_node/chain_spec.rs

```diff
@@ -110,16 +117,48 @@ fn props() -> Properties {
 
 pub fn development_chain_spec(
     genesis_config: GenesisConfig,
+    signers: &[Keypair],
+    genesis_balance: U256,
 ) -> Result<DevelopmentChainSpec, String> {
+    let genesis_balance = SubstrateU256::from(genesis_balance).inner().as_u128();
+    let mut balance_map: BTreeMap<AccountId32, u128> = signers
+        .into_iter()
+        .map(|keypair| (Account::from(keypair.clone()).substrate_account(), genesis_balance))
+        .collect();
+    if let Some(alloc) = &genesis_config.alloc {
```

Comment:
I think you already pointed this out offline, but mentioning here to not forget about it:
we are populating this twice for the accounts in `genesis_config.alloc`, right?

Reply:
Not exactlu populating twice, if there is an account that was generated by the generator and also in the alloc it will overwrite the generated value. Have to check what anvil does.